css-view-transitions-1/Overview.bs

```diff
@@ -368,7 +368,7 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 	The <dfn>snapshot viewport</dfn> covers all areas of the window that could potentially display web content.
 	This area is consistent regardless of root scrollbars or interactive widgets.
 
-	Issue: "Interactive widgets" refers to UI described within a property definition in [[css-viewport#interactive-widget-section]].
+	Issue: "Interactive widgets" refers to UI described within a property definition in css-viewport#interactive-widget-section.
```

Comment:
This was hitting a fatal error, linking to a non-existent spec.

Reply:
Yeah, I'm aware of it. The refs database folks are looking into it. Can you revert this change? Our spec is correct, and this will stop failing when the db is fixed.

Reply:
Huh, it builds for me via `bikeshed watch`